plugins/repository-s3/build.gradle

```diff
@@ -130,6 +129,7 @@ test {
   // this is tested explicitly in separate test tasks
   exclude '**/RepositoryCredentialsTests.class'
   exclude '**/S3RepositoryThirdPartyTests.class'
+  systemProperty 'aws.region', 'us-east-2'
```

Comment:
Note: Local testing fails as a result of the region not being found. It appears https://github.com/opensearch-project/OpenSearch/pull/7372 relied solely on PR tests and local tests were not run. This is a separate problem as many "flaky" tests would be caught if maintainers beast test locally and only rely on PR testing after local tests fully pass.